src/GraphQL.ApiTests/net50/GraphQL.approved.txt

```diff
@@ -3831,6 +3825,12 @@ namespace GraphQL.Validation.Complexity
         public double TotalImpact { get; }
         public GraphQL.Validation.ValidationContext ValidationContext { get; }
     }
+    public struct FieldComplexityResult
+    {
+        public double ChildImpactMultiplier;
+        public double FieldImpact;
+        public FieldComplexityResult(double fieldImpact, double childImpactMultiplier) { }
+    }
```

Comment:
In this case the struct exists primarily to provide names to what was ValueTuple fields.  Should they be fields or props?

Reply:
The `ValueTuple<T1, T2>` struct uses fields, not properties.

https://learn.microsoft.com/en-us/dotnet/standard/value-tuples

Reply:
If the type is public, I'd make them properties. I don't think the performance impact is noticeable.

Reply:
Updated to props as suggested.